src/mina-signer/tests/client.test.ts

```diff
@@ -1,4 +1,4 @@
-import Client from '../dist/node/mina-signer/mina-signer.js';
+import Client from '../../../dist/node/mina-signer/mina-signer.js';
```

Comment:
This changes the build we use for mina-signer tests from the _build of mina-signer only_ to the _build of o1js + mina-signer_.

We don't want this change: We want to test mina-signer in isolation, in the same version as it is published.

Reply:
Oh, okay. That makes sense. I just tried to run the tests, got a bunch of file not found errors and changed the paths to the files that existed on my machine. I should've put more thought into what was going on with that. I see now it is even documented that you need to build mina-signer for the tests to work. 

How would you feel about adding this:
```bash
if ! [ -f /mina-signer/dist ]
then
  pushd src/mina-signer
    npm run build
  popd
fi
```
to the test script so it just builds mina-signer if you need to?

Reply:
ok good idea to add that!